matveev_burikova/graph.cpp

```diff
@@ -163,71 +199,78 @@ class GraphGenerator {
  private:
   Params params_ = Params(0, 0);
 };
+const std::vector<Graph::VertexId> get_not_connected_vertex_ids(
+    Graph graph,
+    Graph::VertexId vertex_id,
+    Graph::Depth with_depth) {
```

Comment:
Усложнили, передали в метод слишком много ненужной информации
```suggestion
const std::vector<Graph::VertexId> get_unconnected_vertex_ids(
    Graph::VertexId id,
    const std::vector<Graph::VertexId>& ids_list) {
```